### DISCUSSION TITLE
Filtering by rows / filtering a column with multiple arguments

### DISCUSSION THREAD

**Discussion by ghost** [vaexio/vaex#1892] [General]
Hey guys ,

So i ve got a dataset looking like this:
![image](https://user-images.githubusercontent.com/39860343/153230603-04e96ae1-0abb-445b-bf56-86e59290115b.png)

I want to filter it, by keeping only the rows where LAT is one  of this ([34.  , 34.05, 34.1 , 34.15, 34.2 , 34.25, 34.3 , 34.35, 34.4 ,
       34.45, 34.5 , 34.55, 34.6 , 34.65, 34.7 , 34.75, 34.8 , 34.85,
       34.9 , 34.95, 35.  , 35.05, 35.1 , 35.15, 35.2 , 35.25, 35.3 ,
       35.35, 35.4 , 35.45, 35.5 , 35.55, 35.6 , 35.65, 35.7 , 35.75,
       35.8 , 35.85, 35.9 , 35.95, 36.  , 36.05, 36.1 , 36.15, 36.2 ,
       36.25, 36.3 , 36.35, 36.4 , 36.45, 36.5 ])

Any ideas how to do that?

![image](https://user-images.githubusercontent.com/39860343/153230416-d6cb6e52-0eb8-431e-aa33-b8f35bf97792.png)

This doesnt work : /
Best Regards

**Comment by JovanVeljanoski** [comment]
Hey! 

Would something like this work

```python
list_of_values = [34. , 34.05, 34.1 , 34.15, 34.2 , 34.25, 34.3]  # Add as many as you need
df_2 = df[df.LAT.isin(list_of_values)]
```

aside:
The example that you should at the end, there are few potential things wrong:
 - you probably want to use the "or" operator | (since LAT can't simultaneously be 34 AND 34.05
 - be mindful of types (are your values string or float etc.).

**Comment by ghost** [comment]
Jovan,

Big thanx! : )

BR
John
